templates/elasticsearch/template.yaml

```diff
@@ -0,0 +1,912 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: AWS Service Broker - Amazon Elasticsearch
+
+###############################################################################
+# Service Broker MetaData
+###############################################################################
+Metadata:
+  AWS::ServiceBroker::Specification:
+    Version: 1.0
+    Tags:
+    - 'AWS'
+    - 'Elasticsearch'
+    Name: elasticsearch
+    DisplayName: Amazon Elasticsearch
+    LongDescription: Amazon Elasticsearch Service is a fully managed service that makes it easy for you to deploy, secure, and operate Elasticsearch at scale with zero down time.
+    ImageUrl: https://s3.amazonaws.com/awsservicebroker/icons/AmazonRDS_LARGE.png
+    DocumentationUrl: https://aws.amazon.com/elasticsearch-service/
+    ProviderDisplayName: Amazon Web Services
+    ServicePlans:
+      production:
+        DisplayName: Production
+        Description: Configuration designed for production deployments
+        LongDescription: Creates an Amazon Elasticsearch optimised for production use
+        Cost: https://aws.amazon.com/elasticsearch-service/pricing/?nc=sn&loc=3
+        ParameterValues:          
+          AvailabilityZones: Auto
+          CidrBlocks: Auto
+          CidrSize: 27          
+          ESDomainName: Auto
+          NodeAvailabilityZones: 3
+          NodeInstanceCount: 3          
+          DedicatedMasterInstanceCount: 3          
+          NodeToNodeEncryption: true
+          EncryptionAtRest: true
+      dev:
+        DisplayName: Development
+        Description: Configuration designed for development and testing deployments
+        LongDescription: Creates an Amazon Elasticsearch optimised for development and testing use
+        Cost: https://aws.amazon.com/elasticsearch-service/pricing/?nc=sn&loc=3
+        ParameterValues:          
+          AvailabilityZones: Auto
+          CidrBlocks: Auto
+          CidrSize: 27          
+          ESDomainName: Auto
+          NodeAvailabilityZones: 1          
+          NodeInstanceCount: 1          
+          DedicatedMasterInstanceCount: 0          
+          NodeToNodeEncryption: true
+          EncryptionAtRest: true
+      custom:
```

Comment:
`Description` and `Cost` are required by CF

Reply:
Only see Description missing under 'custom:', so added that in. Cost is there. But let me know if an issue.